index.test.js

```diff
@@ -1003,6 +1003,98 @@ describe('Real-world usage', () => {
         });
       });
     });
+
+    describe('fr', () => {
+      const frEquivalency = new Equivalency()
+        .doesntMatter(Equivalency.UNICODE_NORMALIZATION)
+        .doesntMatter(Equivalency.WHITESPACE_DIFFERENCES)
+        .doesntMatter(Equivalency.fr.CAPITAL_VOWEL_ACCENTS)
+        .doesntMatter(Equivalency.CAPITALIZATION)
```

Comment:
The order is important here, `CAPITAL_VOWEL_ACCENTS` _must_ come before `CAPITALIZATION` when used or it will have no effect.